src/apps/testapps/testPolygonToCellsExperimental.c

```diff
@@ -643,48 +707,105 @@ SUITE(polygonToCells) {
     }
 
     TEST(polygonToCellsPointPolygon) {
-        int64_t numHexagons;
-        t_assertSuccess(H3_EXPORT(maxPolygonToCellsSizeExperimental)(
-            &pointGeoPolygon, 9, CONTAINMENT_CENTER, &numHexagons));
-        t_assert(numHexagons == 1, "got expected estimated size");
-        H3Index *hexagons = calloc(numHexagons, sizeof(H3Index));
-
-        t_assertSuccess(H3_EXPORT(polygonToCellsExperimental)(
-            &pointGeoPolygon, 9, CONTAINMENT_CENTER, hexagons));
-        int64_t actualNumIndexes = countNonNullIndexes(hexagons, numHexagons);
-
-        t_assert(actualNumIndexes == 0, "got expected polygonToCells size");
-        free(hexagons);
+        for (int res = 0; res < MAX_H3_RES; res++) {
+            int64_t numHexagons;
+            t_assertSuccess(H3_EXPORT(maxPolygonToCellsSizeExperimental)(
+                &pointGeoPolygon, res, CONTAINMENT_CENTER, &numHexagons));
+            t_assert(numHexagons >= 1 && numHexagons <= 5,
+                     "got expected estimated size");
```

Comment:
Updated these assertions to expect a reasonable range rather than a specific value

Reply:
Is 5 the actual max that could be contained? I guess it doesn't really matter as long as it's equal or above the actual number that could be contained.

Reply:
I think the max I saw in tests was 4, but 5 seemed reasonable 🤷